src/udev/udev-node.c

```diff
@@ -99,7 +102,8 @@ static int node_symlink(sd_device *dev, const char *node, const char *slink) {
         if (rename(slink_tmp, slink) < 0) {
                 r = log_device_error_errno(dev, errno, "Failed to rename '%s' to '%s': %m", slink_tmp, slink);
                 (void) unlink(slink_tmp);
-        }
+        } else
+                r = 1;
```

Comment:
maybe add comment `/* tell caller we actually did something, i.e. the symlink wasn't in place yet, and we created it now */`